src/browser/components/MattermostView.jsx

```diff
@@ -157,7 +157,33 @@ const MattermostView = createReactClass({
         console.warn(message);
         break;
       case 2:
-        console.error(message);
+        let match = e.message.match(/Not allowed to load local resource:\s*(.+)/);
+        let resURL = "";
+        let isNetworkDrive = false;
+
+        if (match != null) {
+          if (match.length == 2) {
+            resURL = match[1];
+
+            let u = url.parse(resURL);
+
+            // Is it on a network drive?
+            if (u.protocol === 'file:' && u.host) {
+              isNetworkDrive = true;
+            }
+          }
+        }
+
+        // Network drive: Should be allowed.
+        if (isNetworkDrive) {
+          if (!shell.openExternal(decodeURI(resURL))) {
```

Comment:
Just curious, I would like to know a case. Did you test?

- resURL has a whitespace (%20).

Reply:
Yes, I tested %20, and full-width space(%E3%80%80) as well.